src/renderers/native/ReactNativeStackInspector.js

```diff
@@ -58,7 +60,9 @@ if (__DEV__) {
         measure: callback =>
           UIManager.measure(component.getHostNode(), callback),
         props: getHostProps(component),
-        source: component._currentElement && component._currentElement._source,
```

Comment:
This means we’re doing something else that’s bad. How do `null`s end up in owner hierarchy?